mne/io/cnt/cnt.py

```diff
@@ -235,6 +235,8 @@ def _get_cnt_info(input_fname, eog, ecg, emg, misc, data_format, date_format):
             event_bytes = 8
         elif event_type == 2:
             event_bytes = 19
+        elif event_type == 3:
+            event_bytes = 19
```

Comment:
`elif event_type in (2, 3):` or do you see the current version as conceptually cleaner?